examples/tomo/xray_trafo.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2014, 2015 The ODL development group
```

Comment:
I dont see why they are in the repository at all, do we check that some result is equal to these at any point? Why not store the array in that case?